pkg/kubectl/cmd/get/get.go

```diff
@@ -746,7 +746,10 @@ func (o *GetOptions) printGeneric(r *resource.Result) error {
 		errs = append(errs, err)
 	}
 
-	if len(infos) == 0 && o.IgnoreNotFound {
+	if len(infos) == 0 {
+		if !o.IgnoreNotFound {
+			fmt.Fprintln(o.ErrOut, "No resources found.")
```

Comment:
hmm... this is getting output a ton of times in the integration tests... I don't think this is quite right still